Connectors/src/MySql/Properties/launchSettings.json

```diff
@@ -1,4 +1,5 @@
 {
+  "$schema": "https://json.schemastore.org/launchsettings.json",
```

Comment:
I was under the impression IDEs like Visual Studio (and Rider) were supposed to automatically use schemas found on schemastore.org, is it not?

Reply:
You're right. VS and Rider provide IntelliSense without the schema line, so I removed it.